src/main/java/org/jenkinsci/plugins/pipeline/github/PullRequestGroovyObject.java

```diff
@@ -133,7 +135,11 @@ public boolean isLocked() {
 
     @Whitelisted
     public int getMilestone() {
-        return pullRequest.getMilestone().getNumber();
+        if (pullRequest.getMilestone() != null) {
```

Comment:
I think changing the method signature to return `Integer` instead of `int` might be a better approach.

It preserves the groovy idiom of 
```groovy
if (pullRequest.milestone) {
    // we have a milestone..
}
```